### PR TITLE
False positive

### PR DESCRIPTION
**Pull request by Beat-YT**
look at these results, I think the X as a pseudo is not reliable

```
[
    {
        "id": "7199",
        "name": "An Akinator black award",
        "id_base": "346728",
        "proba": "0.208967",
        "description": "The highest award in Akinator",
        "valide_contrainte": "1",
        "ranking": "31",
        "pseudo": "X",
        "picture_path": "partenaire/y/346728__198022888.jpg",
        "corrupt": "0",
        "relative": "0",
        "award_id": "-1",
        "flag_photo": 0,
        "absolute_picture_path": "https://photos.clarinea.fr/BL_48_en/partenaire/y/346728__198022888.jpg",
        "nsfw": true
    },
    {
        "id": "269",
        "name": "A chip",
        "id_base": "611",
        "proba": "0.0892586",
        "description": "-",
        "valide_contrainte": "1",
        "ranking": "19",
        "pseudo": "X",
        "picture_path": "partenaire/q/611__932816942.jpg",
        "corrupt": "1",
        "relative": "0",
        "award_id": "-1",
        "flag_photo": 0,
        "absolute_picture_path": "https://photos.clarinea.fr/BL_48_en/partenaire/q/611__932816942.jpg",
        "nsfw": true
    },
    {
        "id": "5493",
        "name": "Jana Cova",
        "id_base": "356518",
        "proba": "0.0101962",
        "description": "Porn actress",
        "valide_contrainte": "0",
        "ranking": "533",
        "pseudo": "none",
        "picture_path": "partenaire/k/356518__1490084782.jpg",
        "corrupt": "1",
        "relative": "0",
        "award_id": "-1",
        "flag_photo": 0,
        "absolute_picture_path": "https://photos.clarinea.fr/BL_25_en/partenaire/k/356518__1490084782.jpg",
        "nsfw": false
    }
]
```

### CODE REVIEW

**Comment by Beat-YT** [comment]
`add continue` to fix #126

**Comment by Beat-YT** [comment]
@jgoralcz please review it

**Comment by Beat-YT** [comment]
@jgoralcz

**Comment by jgoralcz** [comment]
so I think it's better safe than sorry if there are some false positives.

**Comment by Beat-YT** [comment]
Look at the results. The is a Pornstar that is not marked as nsfw

**Comment by Beat-YT** [comment]
Pseudo as a X doesn't mean anything

File: src/Akinator.ts [hunk 1]
```diff
@@ -161,7 +199,7 @@ export default class Akinator {
       this.answers = (parameters.elements || []).map((ele) => ele.element);
 
       for (let i = 0; i < this.answers.length; i += 1) {
-        this.answers[i].nsfw = ['x', 'pornstar'].includes((this.answers[i].pseudo || '').toLowerCase());
+        this.answers[i].nsfw = this.answers[i].pseudo.toLowerCase() == 'pornstar' || this.answers[i].description.toLowerCase().includes('porn');
```
[the remark this answers is not in the document]
Comment (jgoralcz) [reply src/Akinator.ts:202]: I suppose you are correct in this sense. If they don't have it labeled, then we shouldn't do anything about it, it's up to the user

File: src/Akinator.ts [hunk 2]
```diff
@@ -95,6 +95,44 @@ export default class Akinator {
     }
   }
 
+  /*
+   * Continue to guess after a "win"(e.g contine guession after a wrong result).
```
[the remark this answers is not in the document]
Comment (Beat-YT) [reply src/Akinator.ts:99]: Nope

Comment (jgoralcz) [inline src/Akinator.ts:99]: what is guession

File: src/Akinator.ts [hunk 3]
```diff
@@ -95,6 +95,44 @@ export default class Akinator {
...
+   */
+  async continue(): Promise<void | Error> {
+    if (!this.uri || !this.urlApiWs) throw new Error(this.noUri);
+    if (!this.uriObj || !this.session || !this.signature) throw new Error(this.noSession);
+
+    const query = new URLSearchParams();
```
Comment (jgoralcz) [inline src/Akinator.ts:105]: can we add this to the other functions and maybe put it in a function call?

**Review by jgoralcz — CHANGES_REQUESTED** [review]

File: src/Akinator.ts (hunk 2 above)
Comment (Beat-YT) [reply src/Akinator.ts:99]: Sorry, my bad

Comment (Beat-YT) [reply src/Akinator.ts:99]: It's like, continue to play

**Comment by Beat-YT** [comment]
I found out that when child mode is enabled, `valide_contrainte` is `0` when its inappropriate

**Comment by Beat-YT** [comment]
@jgoralcz

**Comment by Beat-YT** [comment]
Can you check it? its been a little while.

File: src/Akinator.ts (hunk 2 above)
Comment (jgoralcz) [reply src/Akinator.ts:125]: guession is not a word

Comment (Beat-YT) [reply src/Akinator.ts:125]: ok

**Comment by Beat-YT** [comment]
What we could do is put child mode alway on so that nsfw will work.

File: src/Akinator.ts (hunk 3 above)
Comment (Beat-YT) [reply src/Akinator.ts]: sure

**Comment by Beat-YT** [comment]
@jgoralcz hello?

**Review by Beat-YT — COMMENTED** [review]
Fixed that

File: src/Akinator.ts [hunk 4]
```diff
@@ -1,55 +1,74 @@
...
 
 interface AkinatorConstructor {
   region: region
   childMode?: boolean
   proxyOptions?: string | HttpsProxyAgentOptions;
 }
 
-const answers = [0, 1, 2, 3, 4] as const;
-type answerID = (typeof answers)[number];
+export enum answers {
```
[the remark this answers is not in the document]
Comment (jgoralcz) [reply src/Akinator.ts:24]: actually I think this is fine

**Review by jgoralcz — APPROVED** [review]
lgtm

**Pull request merged**